csrc/python_frontend/fusion_record.h

```diff
@@ -2363,8 +2365,8 @@ struct RandomOpRecord : RecordFunctor {
   }
 
   void operator()(FusionState& fd) final {
-    auto arg1 = fd.getFusionState(args_.at(0).index)->template as<TensorView>();
-    auto arg2 = fd.getFusionState(args_.at(1).index)->template as<TensorView>();
+    auto arg1 = fd.getFusionState(args_.at(0).index)->template as<Val>();
```

Comment:
nitpick, getFusionState already returns a `Val*`, we don't need to call `as` here.